src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -198,7 +198,24 @@
 			Canvas.ZIndex="0"
 			EmptyTextType="{x:Bind ParentShellPageInstance.FilesystemViewModel.EmptyTextType, Mode=OneWay}" />
 
-		<TeachingTip
+        <Border>
+            <StackPanel
+            x:Name="SearchUnindexedItemsPanel"
+			HorizontalAlignment="Center"
+            VerticalAlignment="Bottom"
```

Comment:
`x:name` and `VerticalAligment` should be aligned with the others.

Reply:
I think I fixed it, let me know if there's still a problem